src/app/employer/(home)/[viewId]/page.tsx

```diff
@@ -35,16 +32,15 @@ const JobDetailsPage = ({ params: { viewId } }: { params: { viewId: string } })
             setIsLoading(true);
             try {
                 const response = await axios.get(
-                    `https://carrerhub-backend.vercel.app/api/v1/job/${viewId}`
+                    `https://carrerhub-backend.vercel.app/api/v1/job/${viewId}`,
+                    { withCredentials: true }
```

Comment:
donot harcode the url, extract it to a promise and use it with tanstack/react-query